Python 3.14t (free-threaded) has no binaries

**Describe the bug**
No binaries for Python 3.14t
**To Reproduce**
Use python 3.14t-64 and run:
`pip install simpleitk`
It tries to build source and sometimes you get an error of build.

1. Operating system, version, and architecture
 - OS: Windows 11 25H2
 - Architecture: amd64
2. Programming language, and version [e.g. Python 2.7.14, R 3.2.3].
- Python 3.14t-64
3. Version of SimpleITK 2.5.3, output of the SimpleITK Version() function.
4. How was SimpleITK installed
 - built it from source

**Expected behavior**
A clear installation.

**Additional context**
Some part of log:
```bash
...
CMake Error at CMakeLists.txt:105 (message):
            ITK source code directory path length is too long (64 > 50).Please move the
            ITK source code directory to a directory with a shorter path.
...
```
<details>
  <summary>Full Log</summary>
  
```bash
PS D:\TEMP> pip install simpleitk
Collecting simpleitk
  Using cached simpleitk-2.5.3.tar.gz (2.1 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: simpleitk
  Building wheel for simpleitk (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for simpleitk (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [361 lines of output]
      *** scikit-build-core 0.12.2 using CMake 4.1.1 (wheel)
      *** Configuring CMake...
      loading initial cache file D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeInit.txt
      -- Building for: Visual Studio 18 2026
      -- Selecting Windows SDK version 10.0.26100.0 to target Windows 10.0.26200.
      -- SuperBuild -   SimpleITK_SOURCE_DIR: D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK
      -- SuperBuild - SimpleITK-superbuild
      -- SuperBuild -   SimpleITK_DIR: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/SimpleITK-build
      -- SuperBuild -   ITK_DIR: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ITK-prefix/lib/cmake/ITK
      -- SuperBuild -   SWIG_EXECUTABLE: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/swigwin/swig.exe
      -- SuperBuild - Searching for python
      -- SuperBuild -   PYTHON_INCLUDE_DIR: D:/Users/Sergey/AppData/Local/Python/pythoncore-3.14t-64/Include
      -- SuperBuild -   PYTHON_LIBRARY: D:/Users/Sergey/AppData/Local/Python/pythoncore-3.14t-64/libs/python314t.lib
      -- SuperBuild -   PYTHON_EXECUTABLE: D:/Users/Sergey/AppData/Local/Python/pythoncore-3.14t-64/python.exe
      -- SuperBuild - SimpleITK-python => Requires SimpleITK-superbuild
      -- SuperBuild -   SimpleITK_PYTHON_DIR: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/SimpleITK-python
      -- SuperBuild - SimpleITKPythonPackage => Requires SimpleITK-python
      -- Configuring done (0.8s)
      -- Generating done (0.1s)
      -- Build files have been written to: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build
      *** Building project with Visual Studio 18 2026...
      Р’РµСЂСЃРёСЏ MSBuild 18.0.5+e22287bf1 РґР»СЏ .NET Framework

      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\ZERO_CHECK.vcxproj]
        1>Checking Build System
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
        Creating directories for 'SimpleITK-superbuild'
        Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/CMakeLists.txt
        No download step for 'SimpleITK-superbuild'
        No update step for 'SimpleITK-superbuild'
        No patch step for 'SimpleITK-superbuild'
        Performing configure step for 'SimpleITK-superbuild'
        loading initial cache file D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/SimpleITK-superbuild-prefix/tmp/SimpleITK-superbuild-cache-Release.cmake
        -- Selecting Windows SDK version 10.0.26100.0 to target Windows 10.0.26200.
        -- The C compiler identification is MSVC 19.50.35720.0
        -- The CXX compiler identification is MSVC 19.50.35720.0
        -- Detecting C compiler ABI info
        -- Detecting C compiler ABI info - done
        -- Check for working C compiler: D:/IDE/Microsoft Visual Studio/2026/VC/Tools/MSVC/14.50.35717/bin/Hostx64/x86/cl.exe - skipped
        -- Detecting C compile features
        -- Detecting C compile features - done
        -- Detecting CXX compiler ABI info
        -- Detecting CXX compiler ABI info - done
        -- Check for working CXX compiler: D:/IDE/Microsoft Visual Studio/2026/VC/Tools/MSVC/14.50.35717/bin/Hostx64/x86/cl.exe - skipped
        -- Detecting CXX compile features
        -- Detecting CXX compile features - done
        -- Found Git: D:/Utilities/Git/cmd/git.exe (found version "2.53.0.windows.2")
        -- Performing Test Weak Link MODULE -> SHARED (gnu_ld_ignore) - Failed
        -- Performing Test Weak Link MODULE -> SHARED (osx_dynamic_lookup) - Failed
        -- Performing Test Weak Link MODULE -> SHARED (no_flag) - Failed
        -- Passing variable "ITK_BUILD_DEFAULT_MODULES=ON" to ITK external project.
        -- Passing variable "ITK_DEFAULT_THREADER=Platform" to ITK external project.
        -- Passing variable "ITK_USE_BUILD_DIR=OFF" to ITK external project.
        -- Passing variable "Module_GenericLabelInterpolator=ON" to ITK external project.
        -- Passing variable "Module_ITKIOTransformMINC=ON" to ITK external project.
        -- Passing variable "Module_ITKReview=ON" to ITK external project.
        -- Passing variable "Module_LabelErodeDilate=ON" to ITK external project.
        -- Passing variable "Module_SimpleITKFilters=ON" to ITK external project.
        -- Passing variable "BUILD_DOXYGEN=OFF" to SimpleITK external project.
        -- Passing variable "SITK_CMAKE_MINIMUM_REQUIRED_VERSION=3.16.3" to SimpleITK external project.
        -- Passing variable "SITK_CMAKE_POLICY_VERSION=3.16.3...3.20" to SimpleITK external project.
        -- Passing variable "SimpleITK_BUILD_DISTRIBUTE=ON" to SimpleITK external project.
        -- Passing variable "SimpleITK_BUILD_STRIP=ON" to SimpleITK external project.
        -- Passing variable "SimpleITK_GIT_PROTOCOL=https" to SimpleITK external project.
        -- Passing variable "SimpleITK_INSTALL_DOC_DIR=SimpleITK" to SimpleITK external project.
        -- Passing variable "SimpleITK_LUA_EXECUTABLE=D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua/bin/lua" to SimpleITK external project.
        -- Passing variable "SimpleITK_MAX_DIMENSION=5" to SimpleITK external project.
        -- Passing variable "SimpleITK_USE_ELASTIX=OFF" to SimpleITK external project.
        CMake Warning at D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/CMake/sitkSourceVersion.cmake:41 (message):
          Unable to determine source version!



          Please use the git repository or an official source distribution.

        Call Stack (most recent call first):
          D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/Version.cmake:17 (include)
          SuperBuild.cmake:496 (include)
          SuperBuild.cmake:502 (_set_simpleitk_install_path)
          CMakeLists.txt:47 (include)


        -- Configuring done (15.2s)
        -- Generating done (0.3s)
        -- Build files have been written to: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb
        Performing build step for 'SimpleITK-superbuild'
        в•ЁРўв•Ёв•Ўв•¤Рђв•¤Р‘в•Ёв••в•¤Рџ MSBuild 18.0.5+e22287bf1 в•Ёв”¤в•Ёв•—в•¤Рџ .NET Framework

      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\ZERO_CHECK.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
          1>Checking Build System
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\ITK.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
          Creating directories for 'ITK'
          Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/SuperBuild/CMakeLists.txt
          Performing download step (git clone) for 'ITK'
          Cloning into 'ITK'...
          HEAD is now at f51594ad88 ENH: Bump ITK version to 5.4.5
          No update step for 'ITK'
          No patch step for 'ITK'
          Performing configure step for 'ITK'
          Not searching for unused variables given on the command line.
          loading initial cache file D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ITK-build/CMakeCacheInit.txt
          -- Selecting Windows SDK version 10.0.26100.0 to target Windows 10.0.26200.
          -- The CXX compiler identification is MSVC 19.50.35720.0
          -- The C compiler identification is MSVC 19.50.35720.0
          -- Detecting CXX compiler ABI info
          -- Detecting CXX compiler ABI info - done
          -- Check for working CXX compiler: D:/IDE/Microsoft Visual Studio/2026/VC/Tools/MSVC/14.50.35717/bin/Hostx64/x86/cl.exe - skipped
          -- Detecting CXX compile features
          -- Detecting CXX compile features - done
          -- Detecting C compiler ABI info
          -- Detecting C compiler ABI info - done
          -- Check for working C compiler: D:/IDE/Microsoft Visual Studio/2026/VC/Tools/MSVC/14.50.35717/bin/Hostx64/x86/cl.exe - skipped
          -- Detecting C compile features
          -- Detecting C compile features - done
          CMake Error at CMakeLists.txt:105 (message):
            ITK source code directory path length is too long (64 > 50).Please move the
            ITK source code directory to a directory with a shorter path.


          -- Configuring incomplete, errors occurred!
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-mkdir.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-download.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-update.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-patch.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-configure.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-build.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\bf9c2a968f0147669dcbcaf5779685dd\ITK-install.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\54af4909d183d3265e8f544a3cdde7c1\ITK-complete.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\CMakeFiles\ddc804693c150981bafc3009322dce23\ITK.rule;D:\Users\Sergey\AppData\Local\Temp\pip-install-_wxurzq5\simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf\SimpleITK\SuperBuild\CMakeLists.txt' exited with code 1. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\ITK.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\SuperBuildSimpleITKSourceReal.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
          Generating D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/ExternalSource/lua-5.4.7.tar.gz
          -- Fetching "https://simpleitk.s3.amazonaws.com/public/SHA512/98c5c8978dfdf867e37e9eb3b3ec83dee92d199243b5119505da83895e33f10d43c841be6a7d3b106daba8a0b2bd25fe099ebff8f87831dcc55c79c78b97d8b8"
          -- [download 0% complete]
          -- [download 5% complete]
          -- [download 100% complete]
          -- Downloaded object: "D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/Objects/SHA512/98c5c8978dfdf867e37e9eb3b3ec83dee92d199243b5119505da83895e33f10d43c841be6a7d3b106daba8a0b2bd25fe099ebff8f87831dcc55c79c78b97d8b8"
          Generating D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/ExternalSource/swigwin-4.3.0.zip
          -- Fetching "https://simpleitk.s3.amazonaws.com/public/SHA512/3e9f52d719ee9f326849139df59491c95bf915b1898a2e009964d4cd584bd588f006a259027c76e50612cac0504418999a8fcaa2048408981895f67a2a222c46"
          -- [download 0% complete]
          -- [download 100% complete]
          -- Downloaded object: "D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/Objects/SHA512/3e9f52d719ee9f326849139df59491c95bf915b1898a2e009964d4cd584bd588f006a259027c76e50612cac0504418999a8fcaa2048408981895f67a2a222c46"
          Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/SuperBuild/CMakeLists.txt
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\SuperBuildSimpleITKSource.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
          Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/SuperBuild/CMakeLists.txt
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
          Creating directories for 'Lua'
          Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/SuperBuild/CMakeLists.txt
          Performing download step (verify and extract) for 'Lua'
          -- verifying file...
               file='D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/ExternalSource/lua-5.4.7.tar.gz'
          -- verifying file... done
          -- extracting...
               src='D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/ExternalSource/lua-5.4.7.tar.gz'
               dst='D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua-prefix/src/Lua'
          -- extracting... [tar xf]
          -- extracting... [analysis]
          -- extracting... [rename]
          -- extracting... [clean up]
          -- extracting... done
          No update step for 'Lua'
          Performing patch step for 'Lua'
          Performing configure step for 'Lua'
          Not searching for unused variables given on the command line.
          loading initial cache file D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua-prefix/src/Lua-build/CMakeCacheInit.txt
          -- Selecting Windows SDK version 10.0.26100.0 to target Windows 10.0.26200.
          -- The C compiler identification is MSVC 19.50.35720.0
          -- Detecting C compiler ABI info
          -- Detecting C compiler ABI info - done
          -- Check for working C compiler: D:/IDE/Microsoft Visual Studio/2026/VC/Tools/MSVC/14.50.35717/bin/Hostx64/x86/cl.exe - skipped
          -- Detecting C compile features
          -- Detecting C compile features - done
          -- Configuring done (3.2s)
          -- Generating done (0.1s)
          -- Build files have been written to: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua-prefix/src/Lua-build
          Performing build step for 'Lua'
          С‚РҐРёв•ЁРІС‚РҐРёС‚РҐР±С‚РҐРґв•ЁР С‚РҐРґв•ЁРЎС‚РҐРёС‚РҐРҐС‚РҐРґв•ЁРЇ MSBuild 18.0.5+e22287bf1 С‚РҐРёС‚Р¤РґС‚РҐРёС‚РҐР§С‚РҐРґв•ЁРЇ .NET Framework

      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\ZERO_CHECK.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            1>Checking Build System
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua-prefix/src/Lua/CMakeLists.txt
            lauxlib.c
            lbaselib.c
            lcorolib.c
            ldblib.c
            liolib.c
            lmathlib.c
            loadlib.c
            loslib.c
            lstrlib.c
            ltablib.c
            lutf8lib.c
            linit.c
            lapi.c
            lcode.c
            lctype.c
            ldebug.c
            ldo.c
            ldump.c
            lfunc.c
            lgc.c
            С‚РҐРёв•Ёв–’С‚РҐРёС‚РҐР«С‚РҐРёС‚РҐР¦С‚РҐРёС‚Р¤РґС‚РҐРёС‚Р¦РЎС‚РҐРёС‚РҐР¬С‚РҐРёС‚РҐРҐС‚РҐРёС‚РҐР± С‚РҐРёС‚РҐРЎС‚РҐРёС‚РҐР«С‚РҐРёС‚Р¤РґС‚РҐРёС‚Р¦РЎ...
            С‚РҐРёв•ЁРєС‚РҐРёС‚РҐР«С‚РҐРёС‚РҐР­С‚РҐРёС‚Р¤Р С‚РҐРёС‚РҐРҐС‚РҐРёС‚РҐР§С‚РҐРґв•ЁРЇС‚РҐРґв•ЁР¦С‚РҐРёС‚РҐРҐС‚РҐРґв•ЁРЇ...
            llex.c
            lmem.c
            lobject.c
            lopcodes.c
            lparser.c
            lstate.c
            lstring.c
            ltable.c
            ltm.c
            lundump.c
            lvm.c
            lzio.c
            С‚РҐРёв•Ёв–’С‚РҐРёС‚РҐР«С‚РҐРёС‚РҐР¦С‚РҐРёС‚Р¤РґС‚РҐРёС‚Р¦РЎС‚РҐРёС‚РҐР¬С‚РҐРёС‚РҐРҐС‚РҐРёС‚РҐР± С‚РҐРёС‚РҐРЎС‚РҐРёС‚РҐР«С‚РҐРёС‚Р¤РґС‚РҐРёС‚Р¦РЎ...
            lua.vcxproj -> D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\lib\Release\lua.lib
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\luaexec.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua-prefix/src/Lua/CMakeLists.txt
            lua.c
            luaexec.vcxproj -> D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\bin\Release\lua.exe
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\ALL_BUILD.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua-prefix/src/Lua/CMakeLists.txt
          Performing install step for 'Lua'
          С‚РҐРёв•ЁРІС‚РҐРёС‚РҐР±С‚РҐРґв•ЁР С‚РҐРґв•ЁРЎС‚РҐРёС‚РҐРҐС‚РҐРґв•ЁРЇ MSBuild 18.0.5+e22287bf1 С‚РҐРёС‚Р¤РґС‚РҐРёС‚РҐР§С‚РҐРґв•ЁРЇ .NET Framework

      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\ZERO_CHECK.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            1>Checking Build System
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            lua.vcxproj -> D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\lib\Release\lua.lib
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\luaexec.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            luaexec.vcxproj -> D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\bin\Release\lua.exe
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\ALL_BUILD.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua-prefix\src\Lua-build\install.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Lua.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
            1>
            -- Install configuration: "Release"
            -- Installing: D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/Lua/bin/lua.exe
          Completed 'Lua'
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(558,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\sitk-sb\Swig.vcxproj] [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]
          Creating directories for 'Swig'
          Building Custom Rule D:/Users/Sergey/AppData/Local/Temp/pip-install-_wxurzq5/simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf/SimpleITK/SuperBuild/CMakeLists.txt
          Performing download step (verify and extract) for 'Swig'
          -- verifying file...
               file='D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/ExternalSource/swigwin-4.3.0.zip'
          -- verifying file... done
          -- extracting...
               src='D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/ExternalData/ExternalSource/swigwin-4.3.0.zip'
               dst='D:/Users/Sergey/AppData/Local/Temp/tmpgp_q7xzf/build/sitk-sb/swigwin'
          -- extracting... [tar xf]
          -- extracting... [analysis]
          -- extracting... [rename]
          -- extracting... [clean up]
          -- extracting... done
          No update step for 'Swig'
          No patch step for 'Swig'
          No configure step for 'Swig'
          No build step for 'Swig'
          No install step for 'Swig'
          Completed 'Swig'
      D:\IDE\Microsoft Visual Studio\2026\MSBuild\Microsoft\VC\v180\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-mkdir.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-download.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-update.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-patch.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-configure.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-build.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\29029e803ae88bde70accd12b458f91a\SimpleITK-superbuild-install.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\c605ffcf73d287c8ee7cd512032db717\SimpleITK-superbuild-complete.rule;D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\CMakeFiles\77a21a0bfe8c05ad332b3497527aa0da\SimpleITK-superbuild.rule;D:\Users\Sergey\AppData\Local\Temp\pip-install-_wxurzq5\simpleitk_ee57b0c37a6e49ce93a98e2e976a84bf\CMakeLists.txt' exited with code 1. [D:\Users\Sergey\AppData\Local\Temp\tmpgp_q7xzf\build\SimpleITK-superbuild.vcxproj]

      *** CMake build failed
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for simpleitk
Failed to build simpleitk
error: failed-wheel-build-for-install

× Failed to build installable wheels for some pyproject.toml based projects
╰─> simpleitk
PS D:\TEMP
```
</details>

**PS**
I figured out with changing building folder with
```pwsh
$Env:TEMP = $Env:TMP = "D:\TEMP\tmp"
```
But it's very long process. Please, add binaries. It took 2 hours.
```pwsh
...
  Stored in directory: d:\users\sergey\appdata\local\pip\cache\wheels\d7\c1\7d\65a6f51f91d3d96f99a223b8108619098d288df308832f19d9
Successfully built simpleitk
Installing collected packages: simpleitk
Successfully installed simpleitk-2.5.3
07:41:58 ~#@❯ Yes!   ❮  1h 53m 39s 537ms   
```

--- a/.github/actions/package_csharp/scripts/win_build_csharp.sh
+++ b/.github/actions/package_csharp/scripts/win_build_csharp.sh
@@ -29,7 +29,6 @@ export CXX=cl.exe
 ctest -D dashboard_source_config_dir="Wrapping/CSharp" \
       -D "dashboard_track:STRING=Package" \
       -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-csharp" \
-      -D "CTEST_CMAKE_GENERATOR:STRING=Ninja" \
       -S "${CTEST_SOURCE_DIRECTORY}/.github/workflows/github_actions.cmake" -VV -j 2 || \
       echo "::warning file=win_build_csharp.sh:: There was a build or testing issue with csharp."
 

--- a/.github/actions/package_java/scripts/win_build_java.sh
+++ b/.github/actions/package_java/scripts/win_build_java.sh
@@ -29,7 +29,6 @@ export CXX=cl.exe
 ctest -D dashboard_source_config_dir="Wrapping/Java" \
       -D "dashboard_track:STRING=Package" \
       -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-java" \
-      -D "CTEST_CMAKE_GENERATOR:STRING=Ninja" \
       -S "${CTEST_SOURCE_DIRECTORY}/.github/workflows/github_actions.cmake" -VV -j 2 || \
       echo "::warning file=win_build_csharp.sh:: There was a build or testing issue with Java."
 

--- a/.github/actions/package_python/scripts/mac_build_python.sh
+++ b/.github/actions/package_python/scripts/mac_build_python.sh
@@ -9,7 +9,7 @@ echo "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}"
 
 which python
 python --version
-PYTHON_VERSION=$(python -c 'import sys;print ("{0}{1}".format(sys.version_info[0], sys.version_info[1]))')
+PYTHON_ABI_TAG=$(python -c 'import sysconfig; print(sysconfig.get_config_var("SOABI"))')
 
 Python_EXECUTABLE=$(which python)
 SimpleITK_Python_EXECUTABLE=${Python_EXECUTABLE}
@@ -64,11 +64,11 @@ CMAKE_OSX_ARCHITECTURES:STRING=${CMAKE_OSX_ARCHITECTURES}"
 fi
 
 export CTEST_CACHE
-export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/py${PYTHON_VERSION}"
+export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/${PYTHON_ABI_TAG}"
 
 ctest -D dashboard_source_config_dir="Wrapping/Python" \
       -D "dashboard_track:STRING=Package" \
-      -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-py${PYTHON_VERSION}" \
+      -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-${PYTHON_ABI_TAG}" \
       -S "${CTEST_SOURCE_DIRECTORY}/.github/workflows/github_actions.cmake" -VV -j 2
 
 cmake --build "${CTEST_BINARY_DIRECTORY}" --target dist

--- a/.github/actions/package_python/scripts/win_build_python.sh
+++ b/.github/actions/package_python/scripts/win_build_python.sh
@@ -10,7 +10,7 @@ echo "COREBINARYDIRECTORY: ${COREBINARYDIRECTORY}"
 echo "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}"
 
 python --version
-PYTHON_VERSION=$(python -c 'import sys;print ("{0}{1}".format(sys.version_info[0], sys.version_info[1]))')
+PYTHON_ABI_TAG=$(python -c 'import sysconfig; print(sysconfig.get_config_var("SOABI"))')
 Python_EXECUTABLE=$(python -c "import sys; print(sys.executable)")
 echo $Python_EXECUTABLE
 
@@ -34,14 +34,14 @@ EOM
 
 
 export CTEST_CACHE
-export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/py${PYTHON_VERSION}"
+export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/py${PYTHON_ABI_TAG}"
 
 export CC=cl.exe
 export CXX=cl.exe
 
 ctest -D dashboard_source_config_dir="Wrapping/Python" \
       -D "dashboard_track:STRING=Package" \
-      -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-py${PYTHON_VERSION}" \
+      -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-${PYTHON_ABI_TAG}" \
       -D "CTEST_CMAKE_GENERATOR:STRING=Ninja" \
       -S "${CTEST_SOURCE_DIRECTORY}/.github/workflows/github_actions.cmake" -VV -j 2
 

--- a/.github/actions/package_python/scripts/win_build_python.sh
+++ b/.github/actions/package_python/scripts/win_build_python.sh
@@ -34,7 +34,7 @@ EOM
 
 
 export CTEST_CACHE
-export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/py${PYTHON_ABI_TAG}"
+export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/${PYTHON_ABI_TAG}"
 
 export CC=cl.exe
 export CXX=cl.exe
@@ -42,7 +42,6 @@ export CXX=cl.exe
 ctest -D dashboard_source_config_dir="Wrapping/Python" \
       -D "dashboard_track:STRING=Package" \
       -D "CTEST_BUILD_NAME:STRING=${RUNNER_NAME}-${GITHUB_JOB}-${PYTHON_ABI_TAG}" \
-      -D "CTEST_CMAKE_GENERATOR:STRING=Ninja" \
       -S "${CTEST_SOURCE_DIRECTORY}/.github/workflows/github_actions.cmake" -VV -j 2
 
 ( cd ${CTEST_BINARY_DIRECTORY} && cmake --build "${CTEST_BINARY_DIRECTORY}" --config "${CTEST_CONFIGURATION_TYPE}" --target dist )

--- a/.github/workflows/Package.yml
+++ b/.github/workflows/Package.yml
@@ -88,6 +88,9 @@ jobs:
       DASHBOARD_BRANCH_DIRECTORY: "${{ github.workspace }}/SimpleITK-dashboard"
       CTEST_SOURCE_DIRECTORY: "${{ github.workspace }}"
       CTEST_CONFIGURATION_TYPE: "${{ matrix.cmake-build-type }}"
+      CTEST_CMAKE_GENERATOR: "${{ matrix.cmake-generator }}"
+      CTEST_CMAKE_GENERATOR_TOOLSET: "${{ matrix.cmake-generator-toolset }}"
+      CTEST_CMAKE_GENERATOR_PLATFORM: "${{ matrix.cmake-generator-platform }}"
     strategy:
       fail-fast: false
       max-parallel: 5
@@ -171,9 +174,6 @@ jobs:
         env:
           ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS: 2
           CTEST_BINARY_DIRECTORY: "${{ env.COREBINARYDIRECTORY }}"
-          CTEST_CMAKE_GENERATOR: "${{ matrix.cmake-generator }}"
-          CTEST_CMAKE_GENERATOR_TOOLSET: "${{ matrix.cmake-generator-toolset }}"
-          CTEST_CMAKE_GENERATOR_PLATFORM: "${{ matrix.cmake-generator-platform }}"
           CTEST_OUTPUT_ON_FAILURE: 1
           CTEST_CACHE: |
             WRAP_DEFAULT:BOOL=OFF

--- a/.github/workflows/Package.yml
+++ b/.github/workflows/Package.yml
@@ -49,7 +49,7 @@ jobs:
       - name: Build ${{ matrix.dockerfile }}
         shell: bash
         env:
-          PYTHON_VERSIONS: "cp310-cp310"
+          PYTHON_VERSIONS: "cp310-cp310 cp314-cp314t"
           BUILD_CSHARP: ${{ matrix.build_csharp }}
           BUILD_JAVA: ${{ matrix.build_java }}
           BUILD_PYTHON_LIMITED_API: 1
@@ -126,6 +126,7 @@ jobs:
               ITK_C_OPTIMIZATION_FLAGS:STRING=
               ITK_CXX_OPTIMIZATION_FLAGS:STRING=
               SimpleITK_BUILD_DISTRIBUTE:BOOL=ON
+              CXXFLAGS:STRING= /MP
     steps:
       - uses: actions/checkout@v6
         with:
@@ -224,6 +225,12 @@ jobs:
         uses: ./.github/actions/package_python
         with:
           python-version: "3.10"
+          cmake_osx_architectures: ${{ matrix.cmake_osx_architectures || '' }}
+      - name: Package Python 3.14t
+        uses: ./.github/actions/package_python
+        with:
+          python-version: "3.14t"
+          continue-on-error: true
           cmake_osx_architectures: ${{ matrix.cmake_osx_architectures || '' }}
 
       - name: Upload Artifacts

--- a/Utilities/Distribution/manylinux/imagefiles/cmd.sh
+++ b/Utilities/Distribution/manylinux/imagefiles/cmd.sh
@@ -139,10 +139,12 @@ if [[ ! -z ${BUILD_PYTHON_LIMITED_API:+x} && "${BUILD_PYTHON_LIMITED_API}" -ne 0
     unset USE_LIMITED_API
 fi
 
-
-
 for PYTHON in ${PYTHON_VERSIONS}; do
     Python_EXECUTABLE=/opt/python/${PYTHON}/bin/python
+    if [ ! -x "${Python_EXECUTABLE}" ]; then
+        echo "WARNING: Python ${PYTHON} not available in this image, skipping." >&2
+        continue
+    fi
     PLATFORM=$(${Python_EXECUTABLE} -c "import sysconfig; print(sysconfig.get_platform())")
     build_simpleitk_python &&
         ( auditwheel repair $(find ${BLD_DIR}-${PYTHON}/ -name *.whl) -w ${OUT_DIR}/wheelhouse/;

